packages/react/src/components/progress-tracker/progress-tracker.test.tsx

```diff
@@ -1,97 +1,89 @@
-import { shallow } from 'enzyme';
 import { findByTestId, getByTestId } from '../../test-utils/enzyme-selectors';
-import { renderWithProviders } from '../../test-utils/renderer';
+import { mountWithTheme, renderWithProviders } from '../../test-utils/renderer';
 import { range } from '../../utils/range';
-import { Progress, ProgressTracker } from './progress-tracker';
+import { ProgressTracker, ProgressTrackerStep } from './progress-tracker';
 
-function generateSteps(count: number): ProgressTracker[] {
+function generateSteps(count: number): ProgressTrackerStep[] {
     return range(1, count).map((i) => ({ label: `Step ${i}` }));
 }
 
 describe('Progress Component', () => {
-    it('should render all steps', () => {
-        const steps = generateSteps(3);
-
-        const wrapper = shallow(<Progress steps={steps} value={1} />);
-        const stepsWrapper = getByTestId(wrapper, 'progress-tracker');
-
-        const allSteps = findByTestId(stepsWrapper, 'progress-step-', '^');
-        expect(allSteps.length).toBe(3);
+    beforeEach(() => {
+        jest.spyOn(console, 'warn').mockImplementation(() => { });
     });
 
     it('should render all steps labels', () => {
         const steps = generateSteps(3);
 
-        const wrapper = shallow(<Progress steps={steps} value={1} />);
-        const stepsWrapper = getByTestId(wrapper, 'progress-tracker');
-
-        const allStepsLabels = findByTestId(stepsWrapper, 'progress-step-', '^').map((w) => w.text());
+        const wrapper = mountWithTheme(<ProgressTracker steps={steps} value={1} />);
+        const allStepsLabels = findByTestId(wrapper, 'progress-tracker-step-', '^')
+            .map((w) => getByTestId(w, 'progress-tracker-label').text());
```

Comment:
Ça faut pour tous les tests, mais j'aurais gardé la ligne vide après le wrapper.
```suggestion
        const wrapper = mountWithTheme(<ProgressTracker steps={steps} value={1} />);

        const allStepsLabels = findByTestId(wrapper, 'progress-tracker-step-', '^')
            .map((w) => getByTestId(w, 'progress-tracker-label').text());
```
Ça permet de mieux découper visuellement le given/when/then et d'identifier que ce que tu testes finalement c'est le render initial.